src/main/scala/pl/multiplex/models/Consumer.scala

```diff
@@ -0,0 +1,23 @@
+package pl.multiplex.models
+
+import com.fasterxml.jackson.annotation.JsonBackReference
+import javax.persistence._
+
+import scala.beans.BeanProperty
+
+@Entity
+class Consumer extends Serializable {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @BeanProperty
+  var id: Long = _
+  @BeanProperty
+  var firstName: String = _
+  @BeanProperty
+  var secondName: String = _
+  @OneToMany(mappedBy = "consumer")
+  @JsonBackReference
```

Comment:
obiekt który jednocześnie służy do persystencji i do zwracania danych na froncie? czy to nie antypattern?

Reply:
Rozumiem, że powinienem to stworzyć jeszcze warstwę DTO, żeby to rozdzielić?